max/python/max/pipelines/architectures/gemma3multimodal/arch.py

```diff
@@ -46,11 +45,8 @@
     },
     pipeline_model=Gemma3_MultiModalModel,
     task=PipelineTask.TEXT_GENERATION,
-    tokenizer=TextTokenizer,
+    tokenizer=TextAndVisionTokenizer,
     default_weights_format=WeightsFormat.safetensors,
     multi_gpu_supported=True,
     rope_type=RopeType.normal,
```

Comment:
```suggestion
    rope_type=RopeType.normal,
    required_arguments={
        "enable_prefix_caching": False,
        "enable_chunked_prefill": False,
    },
```

We do not support prefix_caching and chunked_prefill for VLMs at the moment. 
This fixes the issue with batch_size>1 failing